modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -107,6 +107,14 @@
 					/obj/item/weaponcell/medical/toxin)
 	crate_name = "medicell replacement crate"
 
+/datum/supply_pack/medical/afad
+	name = "Automated First Aid Device"
+	desc = "Someone mildly hurt and it's too much of a bother to manually handle their burns or cuts? Look no further than the AFAD, a state-of-the-art pain-relief device!"
+	cost = CARGO_CRATE_VALUE * 20
+	access = ACCESS_MEDICAL
+	contains = list(/obj/item/gun/medbeam/afad)
+	crate_name = "AFAD crate"
```

Comment:
I'd prefer if it stayed as goodie-only, because it's meant for people to use for space exploration, and just generally personal use, I don't want it to become the meta because it's basically free, even if very slow, healing. It would also bring back the beam-crossing explosions in Medbay as a regular occurrence, which I'd really like to avoid as much as I can.

Reply:
Sounds good! I'm going to throw it back in goodies. The main purpose of this PR is the iRCD anyway so I just won't touch it.